pom.xml

```diff
@@ -353,6 +353,8 @@
             <id>jar-local</id>
             <properties>
                 <packagingType>jar</packagingType>
+                <releases.repo>http://maven.release.ctripcorp.com/nexus/content/repositories/flightrelease</releases.repo>
```

Comment:
remove it, publishing jars should be published to central repository not private repository